src/fiskaltrust.Launcher/ServiceInstallation/LinuxSystemD.cs

```diff
@@ -19,17 +32,29 @@ public override async Task<int> InstallService(string commandArgs, string? displ
             {
                 return -1;
             }
+
+            // Creating a directory if does not exist
+            if (!Directory.Exists(_requiredDirectory))
+            {
+                Directory.CreateDirectory(_requiredDirectory);
+
+                // Change of directory owner
+                await RunProcess("chown", new[] { _serviceUser, _requiredDirectory });
+
+                // Changing directory permissions
+                await RunProcess("chmod", new[] { "700", _requiredDirectory });
```

Comment:
Actually giving the user group the same permissions is probably a good idea.. and I don't see a problem with others having read permissions 🤔 I'd probably go with 774 ...
```suggestion
                await RunProcess("chmod", new[] { "774", _requiredDirectory });
```

Reply:
okay, gonna change it :) thanks Paul